src/components/Chat/ChatBody.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useRef } from 'react';
+import Message from './Message';
+import MessageDate from './MessageDate';
+
+type MessageType = {
+  isSender: boolean;
+  userId: string;
+  content: string;
+  timestamp: string;
+};
+
+type UserType = {
+  userId: string;
+  userName: string;
+};
+
+interface Props {
+  messages: MessageType[];
+  users: { [key: string]: UserType };
+}
+
+export default function ChatBody({ messages, users }: Props) {
+  let lastDate: string | '' = '';
```

Comment:
lastDate에 대한 타입으로 string과 ""를 지정해주셨는데 "" 도 결국 string 이기 때문에 string만 타입으로 지정해주셔도 좋을 것 같습니다 !

```suggestion
  let lastDate: string  = '';
```